src/wp-admin/setup-config.php

```diff
@@ -223,7 +223,14 @@ function setup_config_display_header( $body_classes = array() ) {
 		</tr>
 		<tr>
 			<th scope="row"><label for="pwd"><?php _e( 'Password' ); ?></label></th>
-			<td><input name="pwd" id="pwd" type="text" aria-describedby="pwd-desc" size="25" placeholder="<?php echo htmlspecialchars( _x( 'password', 'example password' ), ENT_QUOTES ); ?>" autocomplete="off" /></td>
+			<td>
+				<span class="wp-pwd">
+					<input name="pwd" id="pwd" type="password" class="regular-text" data-reveal="1" data-pw="<?php echo esc_attr( get_option( '' ) ); ?>" aria-describedby="pwd-desc" size="25" placeholder="<?php echo htmlspecialchars( _x( 'password', 'example password' ), ENT_QUOTES ); ?>" autocomplete="off" />
```

Comment:
I'm not sure this `data-pw` even needs to be there, does it? I'll confess that some of these may have been copied from 2070 or 2071 and I just didn't review it close enough. From what I can tell, it can be removed from `setup-config.php`.

Reply:
I removed the `data-pw`, as it seems unused. #2071 also did not include this.